admin/database/database.go

```diff
@@ -333,6 +333,7 @@ type Organization struct {
 	PaymentCustomerID                   string    `db:"payment_customer_id"`
 	BillingEmail                        string    `db:"billing_email"`
 	CreatedByUserID                     *string   `db:"created_by_user_id"`
+	CachedPlanDisplayName               *string   `db:"cached_plan_display_name"`
```

Comment:
1. Keep billing-related fields together, i.e. put the plan name above `CreatedByUserID`
2. See other comment, I don't think a `*string` is warranted here – an empty string should be a sufficient indicator that there's no plan.

Reply:
1. ok
2. based on discussion [above](https://github.com/rilldata/rill/pull/6338#discussion_r1952690650) we should keep this `*string`